.github/workflows/publish_alpha.yml

```diff
@@ -0,0 +1,122 @@
+# This workflow will generate a distribution and upload it to PyPI
+
+name: Publish Alpha Build ...aX
+on:
+  workflow_call:
+    inputs:
+      branch:
+        type: string
+        default: ${{ github.ref }}
+      version_file:
+        type: string
+        default: version.py
+      locale_folder:
+        type: string
+        default: "locale"
+
+jobs:
+  autotranslate:
+    runs-on: ubuntu-latest
+    env:
+      API_KEY: ${{secrets.DL_API_KEY}} 
+      LOCALE_FOLDER: ${{ github.workspace }}/action/package/${{ inputs.locale_folder }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

Comment:
v4 (this is the last one I'll mark, but I suggest using this version in all these workflows)